torchaudio/models/wavernn.py

```diff
@@ -324,3 +349,27 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def get_pretrained_wavernn(checkpoint_name: str, progress: bool = True) -> WaveRNN:
```

Comment:
Can we simply call this `wavernn`?

I am not a fun of progress bar as it often messes up the log when running the code in an non-interactive way, while providing almost no benefit. Do you think it is mission-critical?

Reply:
> Can we simply call this wavernn?
Sure!

For the progress bar, I am okay with removing them. With the model size and the average internet speed, downloading the model won't let most of the users feel like the program is dead. But it may still provide a bit of function for people with slow internet.

Reply:
I've removed the ``progressbar`` for now [here](https://github.com/pytorch/audio/pull/1612/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R370). Please let me know if you think we should add it back and set. the default to False instead.